libraries/radwordsprocessing/model/floatingimage.md

```diff
@@ -137,25 +131,36 @@ The __FloatingImage__ element exposes the following properties:
 
 This section explains the behavior of the  __Size__ property of The __Image__ object in a __FloatingImage__  and how the image sizes are exported to the supported formats.
 
->note Currently, the PdfFormatProvider **does not support floating images** and these elements are skipped when exporting to **PDF**.
+
 
 * __Insert image without specifying its size__
 	* Model: Size will be Size.Empty.
 	* Export to DOCX: Size will be decoded
 	* Export to RTF: Size will be decoded
 	* Export HTML: Width and height attributes will not be exported
+	* Export to PDF: Size will be decoded
 
 
 * __Insert image and set its size to Size.Empty__
 	* Model: Size will be Size.Empty
 	* Export to DOCX: Size will be (1,1)
 	* Export to RTF: Size will be (1,1)
 	* Export to HTML: Width and height attributes will not be exported
+	* Export to PDF: Size will be (1,1)
 
 * __Inserting image without specifying its size and obtain its size later (through the property getter)__
 	* Model: Size will be decoded
 
+## Exporting to PDF
+
+WordsProcessing enables you exporting documents with floating images to PDF. However, the set of supported properties is **limited** compared to the other document formats. Following is a list of the properties that are not supported by the **PdfFormatProvider** class:
```

Comment:
I am not quite sure but I think it should be: _WordsProcessing enables you to export documents..._